recipes/openssl/1.x.x/conanfile.py

```diff
@@ -791,8 +794,11 @@ def _patch_install_name(self):
 
     def _replace_runtime_in_file(self, filename):
         for e in ["MDd", "MTd", "MD", "MT"]:
-            tools.replace_in_file(filename, "/%s " % e, "/%s " % msvc_runtime_flag(self), strict=False)
-            tools.replace_in_file(filename, "/%s\"" % e, "/%s\"" % msvc_runtime_flag(self), strict=False)
+            flag = msvc_runtime_flag(self)
+            if self._is_clangcl and not flag:
+                flag = self.settings.get_safe("compiler.runtime")
+            tools.replace_in_file(filename, "/%s " % e, "/%s " % flag, strict=False)
+            tools.replace_in_file(filename, "/%s\"" % e, "/%s\"" % flag, strict=False)
```

Comment:
```suggestion
        runtime = msvc_runtime_flag(self)
        if self._is_clangcl and not runtime:
            # TODO: to remove if min conan version bumped to a version
            #       implementing https://github.com/conan-io/conan/pull/10898
            runtime = self.settings.get_safe("compiler.runtime")
        for e in ["MDd", "MTd", "MD", "MT"]:
            tools.replace_in_file(filename, "/{} ".format(e), "/{} ".format(runtime), strict=False)
            tools.replace_in_file(filename, "/{}\"".format(e), "/{}\"".format(runtime), strict=False)
```

Reply:
Good. But I would still wait will the next conan update in order to make sure, if it will pass the CI again